server/auth_callout.go

```diff
@@ -116,33 +132,32 @@ func (s *Server) processClientOrLeafCallout(c *client, opts *Options) (authorize
 			// Operator mode is who we send the request on unless switching accounts.
 			issuer = acc.Name
 		}
-		// By default issuer needs to match server config or the requesting account in operator mode.
-		if arc.Issuer != issuer {
+
+		// the jwt issuer can be a signing key
+		jwtIssuer := arc.Issuer
+		if arc.IssuerAccount != "" {
```

Comment:
This conditional should be `if isOperatorMode && arc.IssuerAccount != ""`

In non-operator mode there is no validation for signing keys

Reply:
actually - if it is NOT operator mode, and they set an issuer account, it should fail...